applications/MappingApplication/custom_python/add_custom_mappers_to_python.cpp

```diff
@@ -158,12 +158,12 @@ void ExposeMapperToPython(pybind11::module& m, const std::string& rName)
             ;
 
     // Adding the flags that can be used for mapping
-    mapper.attr("SWAP_SIGN")           = MapperFlags::SWAP_SIGN;
-    mapper.attr("ADD_VALUES")          = MapperFlags::ADD_VALUES;
-    mapper.attr("REMESHED")            = MapperFlags::REMESHED;
-    mapper.attr("USE_TRANSPOSE")       = MapperFlags::USE_TRANSPOSE;
-    mapper.attr("TO_NON_HISTORICAL")   = MapperFlags::TO_NON_HISTORICAL;
-    mapper.attr("FROM_NON_HISTORICAL") = MapperFlags::FROM_NON_HISTORICAL;
+    KRATOS_REGISTER_IN_PYTHON_FLAG_NAME(mapper, MapperFlags::SWAP_SIGN,           "SWAP_SIGN")
```

Comment:
why passing `mapper`  again ? you can directly add them to  application module  no ?

Reply:
you need to pass the pybind-module to which you want to add the attributes

Reply:
Ofc I could also pass the App-module, but I don't want that

Reply:
that is what .. curious why  not .. then you do not have to do `MappingApplication.Mapper.FLAG`
but simply `MappingApplication.FLAG`